README.md

```diff
@@ -6,4 +6,9 @@
 - run `git update-index --skip-worktree scripts/environment-variables.sh`
 - edit `scripts/environment-variables.sh`
 - run `vagrant up --provision`
-- run `vagrant ssh`
\ No newline at end of file
+- run `vagrant ssh`
+
+## environment-variables.sh
+  - consists of keys, password or any other information that is developer or environment specific
+  - should not be checked in to the repository
```

Comment:
also add `is automatically run each time we run vagrant ssh`
looks good otherwise